physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5727,6 +5755,17 @@ subroutine gfs_stability                                              &
 
           endif          ! end of if (dtv >= 0 ) then loop
 !
+! form drag coefficient
+!
+          if ( varf.gt.varf_min .and. psl_gwd_z0m_factor.gt.0.0 ) then
```

Comment:
can use the locally defined `zero` or else add `_kind_phys` to the 0.0

Reply:
changed